### PR TITLE
fix: view collection on drops should not open new window

### LINKED ISSUE #8492: view collection on drops should not open new window
why it is opening in new window when I click on view collection? 

image

### PR DESCRIPTION
**Pull request by Jarsen136**
## PR Type

  - [x] Bugfix

  ## Context

  - [x] Closes #8492

### CODE REVIEW

**Review by roiLeo — APPROVED** [review]
`_self` is default you can remove the line

File: components/collection/unlockable/UnlockableCollectionInfo.vue [hunk 1]
```diff
@@ -23,7 +23,7 @@
           rounded
           :tag="NuxtLink"
           :to="`/${urlPrefix}/collection/${collectionId}`"
-          target="_blank"
+          target="_self"
```
Comment (Jarsen136) [inline components/collection/unlockable/UnlockableCollectionInfo.vue:26]:
```suggestion
```

**Comment by Jarsen136** [comment]
> `_self` is default you can remove the line

✅

**Comment by exezbcz** [comment]
I don't think it's wrong; if wao wants to change it, let's go.

**Comment by roiLeo** [comment]
> I don't think it's wrong; if wao wants to change it, let's go.

Yup I think we should keep open in new tab since if user have already generate some art (but not yet mint) it will loose the generation once click on link.

**Review by daiagi — APPROVED** [review]

**Comment by roiLeo** [comment]
tenor

**Comment by exezbcz** [comment]
@roiLeo yeah, there is still cmd click


nevermind, small change

**Pull request merged**